src/main/scala/firrtl/passes/memlib/CreateMemoryAnnotations.scala

```diff
@@ -6,6 +6,7 @@ package memlib
 
 import firrtl.stage.Forms
 
+@deprecated("CreateMemoryAnnotations will not take reader: Option[YamlFileReader] as argument since 1.5.", "1.4")
```

Comment:
Do you mean "since 1.6"? Because you are trying to merge this into the `1.5` branch.

Reply:
Since `1.4` was already released, I am not sure if we are allowed to introduce breaking changes in `1.5` of things that were not deprecated when `1.4` was released/

Reply:
Oh my bad, copy & paste directly, I'll fix that now.

Reply:
This PR is for breaking change introduced in #2191.
I think I'm backing port this deprecation to next version of 1.4, and we can merge #2191 to 1.5.

Reply:
I think that sounds great!

Reply:
:)